docs/reference/environment-variables.mdx

```diff
@@ -227,6 +227,11 @@ You can find these credentials on the
 [Algolia Dashboard](https://www.algolia.com/dashboard/api-keys), on the **API
 keys** page from the menu.
 
+### Attraqt
+
+`FRONT_COMMERCE_ATTRAQT_SEARCH_API_KEY`: your attraqt search api key, found in
+your XO console under `Search > API Keys`
```

Comment:
```suggestion
`FRONT_COMMERCE_ATTRAQT_SEARCH_API_KEY`: your Attraqt search API key, found in
your XO console under `Search > API Keys`
```

and same remark if a link can be added

Reply:
I can add a link to the console, but unfortunately can't do anything more than that.

I'll add this one at  least though.